templates/app/controllers/spree/orders_controller.rb

```diff
@@ -77,6 +77,8 @@ def populate
         end
       end
     end
+    deprecate populate: 'please use OrdersContentController#create instead',
```

Comment:
I think we can just remove this. Being a template, deprecating code is not needed.

Reply:
Hehe.. You're right. I'll remove the deprecation notice. TODO.